swap_meet/item.py

```diff
@@ -1,2 +1,29 @@
+import uuid
+
+
 class Item:
-    pass
\ No newline at end of file
+   
+    def __init__(self, id=None, condition=0):
+        if id is None:
+            self.id = uuid.uuid4().int
+        else:
+            self.id = id
```

Comment:
This works just fine! For argument's sake, how might you shorten your code into one line using a ternary operator?